src/Resources/skeleton/crud/templates/new.tpl.php

```diff
@@ -0,0 +1,22 @@
+<?= $helper->getHeadPrintCode('New '.$entity_class_name) ?>
+
+{% block stylesheets %}
+<style>
```

Comment:
style should not be here
I think it would be better to remove it and classes example-*

Reply:
Yes =) just experiment I'll remove this stuff completely :)